src/AppInstallerCommonCore/Public/AppInstallerStrings.h

```diff
@@ -145,4 +145,36 @@ namespace AppInstaller::Utility
 
     // Gets the file name part of the given URI.
     std::filesystem::path GetFileNameFromURI(std::string_view uri);
+
+    // Converts a container to a string representation of it.
+    template <typename T, typename U>
+    std::string ConvertContainerToString(const T& container, U toString)
+    {
+        std::ostringstream strstr;
+        strstr << '[';
+
+        bool firstItem = true;
+        for (const auto& item : container)
+        {
+            if (firstItem)
+            {
+                firstItem = false;
+            }
+            else
+            {
+                strstr << ", ";
+            }
+
+            strstr << toString(item);

```

Comment:
Which type(s) were not being output properly by the `<<`?  I would expect any stringish type we use to work without a coercion function.

Not critical, but I like to understand 😄

Reply:
`Utility::Architecture` for the applicable architectures filter. It was coming out as an integer.